document-picture-in-picture/main.js

```diff
@@ -0,0 +1,90 @@
+
+const videoPlayer = document.getElementById("player");
+const playerContainer = document.getElementById("container");
+let togglePipButton;
```

Comment:
Same for `togglePipButton`. It's not necessary. You can remove it there.

Reply:
Again, updated.